src/kartograf/atom_mapping_scorer.py

```diff
@@ -3,11 +3,5 @@
 
 import logging
 
-from .mapping_metrics import MappingRMSDScorer
```

Comment:
These are indeed unused imports, but they are kept around to avoid breaking API. We will remove them in v2.0 unless they are a big issue for others (happy to put a noqa or other linter stop here if needed!)

Reply:
the imports in this file, I think it's the only ones causing CI failures

Reply:
done